jscomp/syntax/ast_external_attributes.ml

```diff
@@ -555,7 +555,7 @@ let handle_attributes
           left_attrs
         end
 
-      | _ -> Location.raise_errorf ~loc "conflict attributes found [@@bs.obj]"  
+      | _ -> Location.raise_errorf ~loc "Attribute found that conflicts with [@@bs.obj]"  
```

Comment:
if we want to provide really great error message, it would be tedious work.
for example: here `[@@bs.obj]` is used but `st` has some other conflicting attributes we need check manually which one would be the offensive